install.sh

```diff
@@ -0,0 +1,188 @@
+#!/bin/sh
+
+set -e
+
+DOTENV_LINTER="dotenv-linter"
+DOTENV_LINTER_REPO="${DOTENV_LINTER}/${DOTENV_LINTER}"
+DOTENV_LINTER_GITHUB="https://github.com/${DOTENV_LINTER_REPO}"
+DOTENV_LINTER_RELEASES="${DOTENV_LINTER_GITHUB}/releases"
+
+usage() {
+    require_cmd cat
+    this=$1
+    cat 1>&2 <<EOF
+$this: download binaries for dotenv-linter
+
+USAGE:
+    $this [FLAGS] [OPTIONS] <tag>
+
+FLAGS:
+    -h, --help      Prints help information
+
+OPTIONS:
+    -b, --bindir <DIR_PATH>     Sets bindir or installation directory. Defaults to ./bin
+
+ARGS:
+    <tag>       is a tag from ${DOTENV_LINTER_RELEASES}. If tag is missing, then the latest will be used.
+EOF
+    exit 2
+}
+
+parse_args() {
+  BINDIR=${BINDIR:-./bin}
+  while [ "$#" -gt 0 ]; do
+    case $1 in
+        -h|--help)
+            usage "$0"
+            shift # past argument
+            ;;
+        -b|--bindir)
+            BINDIR="$2"
+            shift # past argument
+            shift # past value
+            ;;
+        *) TAG=$1
+            shift # past argument
+            ;;
+    esac
+  done
+}
+
+main() {
+    parse_args "$@"
+
+    require_cmd uname
+    require_cmd mktemp
+    require_cmd grep
+    require_cmd rm
+
+    get_architecture || return 1
+    _arch="$RETVAL"
+
+    case $_arch in
+        win-*) _ext=".zip" ;;
+        *) _ext=".tar.gz" ;;
+    esac
+
+    _archive_name="${DOTENV_LINTER}-${_arch}${_ext}"
+
+    if [ -z "${TAG}" ]; then
+        println "The latest version will be installed."
+        _url="${DOTENV_LINTER_RELEASES}/latest/download/${_archive_name}"
+    else
+        println "Version ${TAG} will be installed"
+        _url="${DOTENV_LINTER_RELEASES}/download/${TAG}/${_archive_name}"
+    fi
+
+    # Installation
+    _temp_dir=$(mktemp -d)
+    _archive_path="${_temp_dir}/${_archive_name}"
+
+    download "${_url}" "${_archive_path}" || return 1
+
+    case $_archive_path in
+        *.zip)
+            required_cmd unzip
+            unzip "${_archive_path}" -d "${_temp_dir}"
+            _file_name="${DOTENV_LINTER}.exe"
+        ;;
+        *)  require_cmd tar
+            tar -xzf "${_archive_path}" -C "${_temp_dir}"
+            _file_name="${DOTENV_LINTER}"
+        ;;
+    esac
+
+    test ! -d "${BINDIR}" && install -d "${BINDIR}"
+
+    install "${_temp_dir}/${_file_name}" "${BINDIR}/"
+    println "Successfully installed to ${BINDIR}/${_file_name}"
```

Comment:
It would be nice if we got the version of dotenv-linter and printed it here.
What do you think?